pandas/tests/indexes/test_setops.py

```diff
@@ -2,10 +2,13 @@
 The tests in this package are to ensure the proper resultant dtypes of
 set operations.
 """
+from sys import intern
```

Comment:
what is this for?

Reply:
I think this is vscode being overzealous with it's auto imports, I'll remove it.